bindings/rust/integration/tests/internet_http_client.rs

```diff
@@ -0,0 +1,107 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+/// Purpose: ensure that s2n-tls is compatible with other http/TLS implementations
+///
+/// This test uses s2n-tls-hyper to make http requests over a TLS connection to
+/// a number of well known http sites.
+#[cfg(feature = "network-tests")]
+mod http_get {
```

Comment:
I wonder if it's better to put this entire file in a module, or to just add the feature flag to the test method 🤔

Reply:
I mean, isn't the "vestigial" network.rs the standard way this is done is Rust? Maybe ask @camshaft?